common/djangoapps/static_replace/test/test_static_replace.py

```diff
@@ -196,6 +196,21 @@ def test_regex():
         assert_false(re.match(regex, s))
 
 
+@patch('static_replace.staticfiles_storage', autospec=True)
+@patch('static_replace.modulestore', autospec=True)
+def test_static_url_with_xblock_resource(mock_modulestore, mock_storage):
+    """
+    Make sure that for URLs with XBlock resource URL, which start with /static/,
+    we don't rewrite them.
+    """
+    mock_storage.exists.return_value = False
+    mock_modulestore.return_value = Mock(MongoModuleStore)
+
+    pre_text = 'EMBED src ="/static/xblock/resources/babys_first.lil_xblock/public/images/pacifier.png"'
```

Comment:
The pre-text specifically mimics an anchor tag's `href`, having the double quotes and all.